lib/services/FirestoreUtils.dart

```diff
@@ -50,22 +51,36 @@ class FireStoreUtils {
   Future<Feeder> getFeeder(String fid) async {
     DocumentSnapshot feederDocument = await firestore
         .collection(USERS)
-        .doc(this.userID)
+        .doc(FireStoreUtils.userID)
```

Comment:
For each of these where `this` was changed to `FireStoreUtils` what's up with this?

Reply:
The change to static variables to track the current user, feeder, and pet within scope of the FireStoreUtils required this change. There is no more "this" because these variables are treated like a singleton. There may be another solution to this where the functions themselves are static, and I will explore this.